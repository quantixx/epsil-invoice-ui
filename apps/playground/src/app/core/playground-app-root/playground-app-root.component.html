<!--  -->

<epsil-playground-navbar></epsil-playground-navbar>

<!--  -->

<router-outlet>
</router-outlet>

<!--  -->
