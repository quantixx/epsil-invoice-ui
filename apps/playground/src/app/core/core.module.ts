//

import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { RouterModule } from '@angular/router';
import { NavbarComponent } from './navbar/navbar.component';
import { PlaygroundAppRootComponent } from './playground-app-root/playground-app-root.component';

//

@NgModule({
  declarations: [
    PlaygroundAppRootComponent,
    PlaygroundAppRootComponent,
    NavbarComponent
  ],
  exports: [PlaygroundAppRootComponent],
  imports: [CommonModule, RouterModule]
})
export class CoreModule {}

export { PlaygroundAppRootComponent };
