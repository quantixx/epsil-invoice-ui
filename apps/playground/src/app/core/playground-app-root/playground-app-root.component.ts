//

import { Component } from '@angular/core';

//

@Component({
  selector: 'epsil-playground-app-root',
  styleUrls: ['./playground-app-root.component.scss'],
  templateUrl: './playground-app-root.component.html'
})
export class PlaygroundAppRootComponent {}
