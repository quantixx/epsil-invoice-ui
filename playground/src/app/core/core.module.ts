import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { RouterModule } from '@angular/router';
import { PlaygroundAppRootComponent } from './playground-app-root/playground-app-root.component';

@NgModule({
  imports: [CommonModule, RouterModule],
  declarations: [PlaygroundAppRootComponent, PlaygroundAppRootComponent],
  exports: [PlaygroundAppRootComponent]
})
export class CoreModule {}

export { PlaygroundAppRootComponent };
