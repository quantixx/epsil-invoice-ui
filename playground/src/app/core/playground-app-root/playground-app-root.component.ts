import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'my-playground-app-root',
  templateUrl: './playground-app-root.component.html',
  styleUrls: ['./playground-app-root.component.scss']
})
export class PlaygroundAppRootComponent implements OnInit {

  constructor() { }

  ngOnInit() {
  }

}
