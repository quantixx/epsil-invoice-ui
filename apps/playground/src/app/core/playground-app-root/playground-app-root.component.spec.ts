//

import { TestBed } from '@angular/core/testing';
import { RouterTestingModule } from '@angular/router/testing';
import { MockComponent } from 'ng2-mock-component';
import { PlaygroundAppRootComponent } from './playground-app-root.component';

//

describe('Integration::PlaygroundAppRootComponent', () => {
  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [
        PlaygroundAppRootComponent,
        MockComponent({ selector: 'epsil-playground-navbar' })
      ],
      imports: [RouterTestingModule]
    }).compileComponents();
  });

  it('should create', () => {
    const fixture = TestBed.createComponent(PlaygroundAppRootComponent);
    const component = fixture.componentInstance;

    fixture.detectChanges();

    expect(component).toBeTruthy();
    expect(fixture).toMatchSnapshot();
  });
});
