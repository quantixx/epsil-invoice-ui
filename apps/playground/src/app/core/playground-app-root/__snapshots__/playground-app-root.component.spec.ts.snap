// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Integration::PlaygroundAppRootComponent should create 1`] = `
<epsil-playground-app-root>
  
  
  <epsil-playground-navbar />
  
  
  
  <router-outlet>
    
  
  </router-outlet>
  
  
  
</epsil-playground-app-root>
`;
